### DISCUSSION TITLE
Adjust birdseye threshold for specific camera?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4990] [Ask A Question]
I have a wobbly camera that jitters in the wind so it always shows up in birdseye - is there a way to reduce its threshold for detection?

**Comment by NickM-27** [comment]
Yes you can change the motion threshold, check the config docs

**Comment by thinkloop** [comment]
I saw motion in the general config but not under a specific camera, can I apply it there as well?

**Reply by NickM-27** [reply]
Yes anywhere it says "can be overridden at the camera level" it can be done there too
